kolibri/plugins/learn/assets/src/views/ChannelRenderer/CustomContentRenderer.vue

```diff
@@ -117,6 +117,7 @@
           getParams: {
             ids: options.ids,
             parent: options.parent === 'self' ? this.topic.id : options.parent,
+            ...options.extra,
```

Comment:
I'd rather have these continue to be explicit to keep the API documented, otherwise, we might look for where the `random` filter in the FilterSet is being used and realize we don't have any code references to it anywhere, and then delete it.

Reply:
Yes I agree, all should be explicit and documented. No "extra" parameters. I see the way you found to pass extra parameters as a fantastic starting point, but it's a shortcut. All options should be explicit and well documented. Not directly accessing the backend from the frontend.

From the frontend end, the API could be designed like this for fetching 5 random results using pagination:

```
options: {
  randomize_results: true,
  page_size: 5,
  page: 1,
}
```

Or add a separate API method to fetch N random content nodes, which is all we need. But that would only fit our current needs.